.github/workflows/e2e_user_features.yml

```diff
@@ -24,6 +24,8 @@ jobs:
       - name: Setup python by installing wheel
         if: startsWith(github.head_ref, 'update-changelog-for-release') == false
         run: pip install wheel==0.35.0
+      - name: Priint env
```

Comment:
expected change?

Reply:
Yeah i wanted to log the env for auditing, I'm going remove it in my next commit*